packages/mobile/ios/Podfile.lock

```diff
@@ -552,6 +556,7 @@ DEPENDENCIES:
   - react-native-randombytes (from `../node_modules/react-native-randombytes`)
   - react-native-safe-area-context (from `../node_modules/react-native-safe-area-context`)
   - react-native-static-server (from `../node_modules/react-native-static-server`)
+  - react-native-track-player (from `../node_modules/react-native-track-player`)
   - react-native-version-number (from `../node_modules/react-native-version-number`)
   - react-native-video (from `../node_modules/react-native-video`)
```

Comment:
Can we remove react-native-video now? Looks like maybe not cause of collectibles?

Reply:
looks like its still used in one spot for AutoSizeVideo